Makefile

```
@@ -69,9 +69,13 @@ GO ?= GOCACHE=/tmp/.cache docker run --rm -v "$(CURDIR)":/go/src/github.com/k0sp
 ifeq ($(TARGET_OS),windows)
 build: k0s.exe
 else
-build: k0s
+build: .k0sbuild.docker-image.k0s k0s
 endif
 
+.k0sbuild.docker-image.k0s:
```

```suggestion
.k0sbuild.docker-image.k0s: build/Dockerfile
```

We should regenerate the docker image if the docker file changes